azure-devops/azext_devops/dev/team/arguments.py

```diff
@@ -129,10 +126,8 @@ def load_team_arguments(self, _):
                          for given user/group and token.')
 
     with self.argument_context('devops extension') as context:
-        context.argument('include_built_in', arg_type=get_enum_type(_TRUE_FALSE_SWITCH),
-                         help='Include built in extensions.')
-        context.argument('include_disabled', arg_type=get_enum_type(_TRUE_FALSE_SWITCH),
-                         help='Include disabled extensions.')
+        context.argument('include_built_in', help='Include built in extensions.')
+        context.argument('include_disabled', help='Include disabled extensions.')
```

Comment:
arg_type?

Reply:
Ok.. We are converting this from string to bool now, given the default  in  the function definition.

Reply:
yup..
no action here